src/routes/postTransition.ts

```diff
@@ -50,6 +54,50 @@ const transitionEndpoints = (router: Router, dependencies: IDependencies): void
     authentication,
     dependencies.generalPartnerPostTransitionController.generalPartnerChoice()
   );
+
+  router.get(
```

Comment:
Just wondering why General Partner Person routing is being added here as story only talks about Legal Entity?